### ISSUE TITLE
run_test.py not in pip distribution

### ISSUE THREAD

**Issue by jldohmann** [herobank110/factorygame#6]
Hi, saw your post on DEV and thought this was a cool project! Would love to help out for Hacktoberfest 😄 

I installed via pip, but `run_test.py` I think only comes with cloning the repo. It wasn't clear from the docs, so maybe a good task would be to add more explicit instructions for getting started. I'm happy to do that! Could also add the file to the pip package as well.

**Comment by herobank110** [comment]
Hi, thanks so much for your interest.
The current getting started documentation is certainly unclear about how to run the tests.

I was tentative to include any of the tests in the published package since they weren't namespaced under the factorygame package, but I agree that having access to them would be beneficial for getting started.

Do you think it would be clearer if the documentation had a new section "Explore Demos" under getting started that showed the steps to download the repo and run the tests?

**Comment by jldohmann** [comment]
> Do you think it would be clearer if the documentation had a new section "Explore Demos" under getting started that showed the steps to download the repo and run the tests?

Yes, absolutely. I'll draft something up and submit a PR if that's alright

**Comment by herobank110** [comment]
Thanks 😊
